src/main/java/no/nav/pto/veilarbportefolje/postgres/PostgresQueryBuilder.java

```diff
@@ -145,7 +147,10 @@ public void navnOgFodselsnummerSok(String soketekst) {
             whereStatement.add(FODSELSNR + " LIKE '" + soketekst + "%'");
         } else {
             String soketekstUpper = soketekst.toUpperCase();
-            whereStatement.add("(UPPER(" + FORNAVN + ") LIKE '%" + soketekstUpper + "%' OR UPPER(" + ETTERNAVN + ") LIKE '%" + soketekstUpper + "%')");
+            List<String> nameSplittedBySpaces = splitStringBySpaces(soketekstUpper);
```

Comment:
Hmm should we find a cleaver way to make this work for double names like FORNAVN = "Kari Anne" ?

Reply:
And also if you have time, I read that using UPPER on columns are really inefficient. Maybe we can just ensure that all names are always stored as uppercase in the DB?

Reply:
Hm, I am not sure that upper will influence query, I will test that assumption. I will try to write some tests for double name, just to see how current solution is performing :) Btw, thank you for useful comment!